File: backend/src/main/java/com/sai/backend/domain/ai/dto/response/AiFeedbackResponseDto.java
package com.sai.backend.domain.ai.dto.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
public class AiFeedbackResponseDto {
    private String feedbackId;
    private String feedback;
    private String redirectUrl;
}

File: app/SAI/SAI/SAI.Application/Dto/AiFeedbackResponseDto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SAI.SAI.Application.Dto
{
    public sealed class AiFeedbackResponseDto
    {
        [Required] public string feedbackId { get; set; }
        [Required] public string redirectUrl { get; set; }
        [Required] public string feedback { get; set; }
    }
}
